src/sentry/api/issue_search.py

```diff
@@ -92,6 +94,27 @@ def convert_status_value(value, projects, user, environments):
     return parsed
 
 
+def convert_category_value(value, projects, user, environments):
+    if features.has("projects:performance-issue-details-backend", projects[0]):
```

Comment:
idk if `projects[0]` really makes sense here but I'm not sure how the project level feature flag is generally used.

Reply:
Is `category` a new search criteria we're introducing? Not sure about considerations we have to take in terms of flagr performance and calls, but we can iterate through the projects and if one project is feature-flagged on, we return the valid result. Otherwise, we raise an error.

Reply:
Yeah I'm worried it's potentially a lot of projects to go through, and what if they're selecting multiple projects to search but not all are flagged in?

Reply:
We might be better off with an org level feature flag instead. I don't think we will be enabling this on a project level, since it gives an inconsistent experience across an org.